src/sass/partials/global/_mixins.scss

```diff
@@ -62,11 +62,11 @@
       border: setting-get('input border width') solid setting-get('primary color');
       content: '';
       display: inline-block;
-      height: 1em;
-      margin-right: setting-get('rhythm') / 2;
-      transform: translateY(-.125em);
+      height: 1.25em;
+      margin-right: setting-get('rhythm') / 4;
+      transform: translateX(-.8rem);
```

Comment:
Yes, .8 is arbitrary.  I'll find something that works with our `rhythm` setting as is divisable by all the things.